strongbox-storage/strongbox-storage-api/src/main/java/org/carlspring/strongbox/providers/repository/HostedRepositoryProvider.java

```diff
@@ -129,4 +137,25 @@ public Long count(RepositorySearchRequest searchRequest)
                                                    searchRequest.getCoordinates(), searchRequest.isStrict());
     }
 
+    @Override
+    public RepositoryPath getPath(String storageId,
+                                  String repositoryId,
+                                  String path)
+                 throws IOException,
+                        NoSuchAlgorithmException,
+                        ArtifactTransportException,
+                        ProviderImplementationException
```

Comment:
`throws IOException,
NoSuchAlgorithmException,
ArtifactTransportException,
ProviderImplementationException
`

I doubt all of them are necessary here :)